Tests/EmojiPickerViewControllerTests/EmojiLoaderTests.swift

```diff
@@ -28,25 +28,23 @@ import XCTest
 
 // Emoji Counts: https://unicode.org/emoji/charts/emoji-counts.html
 
-let totalEmojiCounts =            3633
-let cEmojiCounts =                1354
-let cSkinTonedEmojiCounts =       645
-let zHairEmojiCounts =            12
-let zHairSkinTonedEmojiCounts =   60
-let zGenderEmojiCounts =          102
-let zGenderSkinTonedEmojiCounts = 470
-let zRoleEmojiCounts =            60
-let zRoleSkinTonedEmojiCounts =   300
-let zFamilyEmojiCounts =          32
-let zFamilySkinTonedEmojiCounts = 235
-let zSkinTonedEmojiCounts =       65
-let zcEmojiCounts =               13
-let emojiKeycapSequenceCounts =   12
-let emojiFlagSequenceCounts =     258
-let emojiTagSequenceCounts =      3
-let componentCounts =             9
-
-let e14EmojisExclusionCounts =    37
+let totalEmojiCounts =            3633 // Structure
+let cEmojiCounts =                1354 // Ⓒ
+let cSkinTonedEmojiCounts =       645  // Ⓒ ‧ 🟫
+let zHairEmojiCounts =            12   // Ⓩ ‧ 🦰
+let zHairSkinTonedEmojiCounts =   60   // Ⓩ ‧ 🦰 ‧ 🟫
+let zGenderEmojiCounts =          102  // Ⓩ ‧ ♀
+let zGenderSkinTonedEmojiCounts = 470  // Ⓩ ‧ ♀ ‧ 🟫
+let zRoleEmojiCounts =            60   // Ⓩ ‧ 👩
+let zRoleSkinTonedEmojiCounts =   300  // Ⓩ ‧ 👩 ‧ 🟫
+let zFamilyEmojiCounts =          32   // Ⓩ ‧ 👪
+let zFamilySkinTonedEmojiCounts = 235  // Ⓩ ‧ 👪 ‧ 🟫
+let zSkinTonedEmojiCounts =       65   // Ⓩ ‧ 🟫
+let zcEmojiCounts =               13   // Ⓩ ‧ Ⓒ
+let emojiKeycapSequenceCounts =   12   // #️⃣
+let emojiFlagSequenceCounts =     258  // 🏁
+let emojiTagSequenceCounts =      3    // 🏴
+let componentCounts =             9    // 🔗
```

Comment:
Nice comment!!

Reply:
Thank you! I was learning as I was going along, this helped!